integrations/api_oauth2_apps_test.go

```diff
@@ -83,9 +85,6 @@ func testAPIDeleteOAuth2Application(t *testing.T) {
 	oldApp := models.AssertExistsAndLoadBean(t, &models.OAuth2Application{
 		UID:  user.ID,
 		Name: "test-app-1",
-		RedirectURIs: []string{
```

Comment:
reason to remove this?

Reply:
ok because of `testAPIUpdateOAuth2Application` ... you can resolve this

Reply:
Yes exactly, suggestions?